homework/shelkovich_ilya/userValidationService/src/test/java/com/tutrit/java/quickstart/service/UserValidationServiceTest.java

```diff
@@ -0,0 +1,45 @@
+package com.tutrit.java.quickstart.service;
+
+import com.tutrit.java.quickstart.bean.User;
+import com.tutrit.java.quickstart.exceptions.UserValidationException;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class UserValidationServiceTest {
+
+    UserValidationService userValidationService;
+    User user;
+
+    @Before
+    public void setUp() {
+        this.userValidationService = new UserValidationService();
+        this.user = new User();
+    }
+
+    @Test
+    public void shouldThrowValidateExceptionLastName() throws UserValidationException {
+        user.setFirstName("Anna");
+        user.setLastName("Matarazzzzzzzzzio");
+        user.setAge(21);
+        Assert.assertFalse(userValidationService.validate(user));
+
+        user.setFirstName("An");
+        user.setLastName("Matarazo");
+        user.setAge(21);
+        Assert.assertFalse(userValidationService.validate(user));
+
+        user.setFirstName("Anna");
+        user.setLastName("Matarazo");
+        user.setAge(2021);
+        Assert.assertFalse(userValidationService.validate(user));
```

Comment:
I guess this method needs to be split. Or you can use the constructor, then it will be more compact.

Reply:
of fabric method )

Reply:
fixed